docs/billing.md

```diff
@@ -4,35 +4,42 @@ A ConnexCS subscription comes with two main tariffs used for billing: **per minu
 
 ## ConnexCS Billing Per Channel
 
-Billing per-channel is the traditional approach where the consumer pays for a set number of channels or ports. It is similar to the VoIP equivalent of per-line billing in mobile networks. 
+Billing per-channel is the traditional approach, where the consumer pays for a set number of channels or ports. It's like the VoIP equivalent of per-line billing in mobile networks.
 
-### Advantages
+### Advantages of Billing Per Channel
 
-+ Cheaper than per minute billing in systems with consistent traffic and high utilization.
-+ Free servers are available for many tiers of service.
-+ Predictable costs allow for more stable billing cycles and simplified cost/benefit analysis.
++ Cheaper than per-minute billing in systems with consistent traffic and high usage.
++ Free servers are available for several tiers of service.
++ Predictable costs allow for more stable billing cycles and simplified cost-benefit analysis.
 
-### Disadvantages
+### Disadvantages of Billing Per Channel
 
 + Inconsistent traffic often leads to higher costs.
 + Capacity planning is inherently less flexible.
 
 ## ConnexCS Billing per minute
 
-In per-minute billing, the amount of seconds you use are tallied daily, combined with the per minute cost, and then subtracted from your account.
+In per-minute billing, the number of seconds you use gets tallied daily. Its combined with the per-minute cost, and then subtracted from your account.
 
-### Advantages
- 
- + Channels come with peripheral considerations that make it more challenging to scale a billing unit, but per-minute rates are exact enough to scale with the system.
+### Advantages of ConnexCS Billing per minute
 
-### Disadvantages
+1. Channels come with peripheral considerations that make it more challenging to scale a billing unit. But per-minute rates are exact enough to scale with the system.
 
- + More expensive with consistent traffic.
- + Multiple servers are more expensive.
+### Disadvantages of ConnexCS Billing per minute
+
+1. More expensive with consistent traffic.
+2. Multiple servers are more expensive.
 
 ## Minimum Charges
 
-All accounts must spend a minimum of $20 per month to remain active. This helps to prevent incurring losses from stagnant accounts. 
+All accounts must spend a minimum of $20 per month to remain active. This helps to prevent incurring losses from stagnant accounts.
```

Comment:
Please include "$20 Per Month/Per Server" instead of $20 per month

Reply:
Made 2 changes:

1. Please include "$20 Per Month/Per Server" instead of $20 per month
2. Removed "more" from "more expensive"

https://bani-2022-12-27--connexcs-docs.netlify.app/billing/